src/msg/async/AsyncMessenger.h

```diff
@@ -325,7 +325,7 @@ class AsyncMessenger : public SimplePolicyMessenger {
 
   void _init_local_connection() {
     ceph_assert(lock.is_locked());
-    local_connection->peer_addrs = *my_addrs;
```

Comment:
since this local_connection is explicitly constructed, can't we set the bool is_loopback from the constructor?